noxfile.py

```diff
@@ -71,8 +71,10 @@ def docs_live(session):
 def test(session):
     """Run the test suite."""
     if _should_install(session):
-        session.install("-e", ".[test]")
+        session.install("-e", ".[test,doc]")
```

Comment:
on second thought, would it simplify things if `nox -s test` was left as-is, and you create a new session (`nox -s a11y`?) that does the live doc build followed by the playwright stuff?

IMO, the advantage to that approach is that sometimes as a dev you know that the things you're changing won't affect accessibility, but might break other tests, so it would be nice to be able to have an easy way to exclude the a11y test when doing code-test-code-test-code... cycles.

Reply:
I'm +1 for the a11y separate session. 
If you want to run the full thing you can still run `nox` alone and all session will be run sequentialy.

Reply:
I do agree on separating sessions - depending on what one is working on at a time one would only be running one or two sessions at a time
For example if working on docs (let's say narrative only) - unless we have a checker for spelling, or complexity (?) it does not make sense to run the full test suite (at least on local dev, at-merge-time would be a different story)